internal/servicehash/servicehash.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package servicehash
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"sync"
+)
+
+const (
+	maxCacheSize = 1000
```

Comment:
Just curious, where did this cache size come from?

Reply:
Good question, I got this file from the original work that Barbayar was doing. @Barbayar any info on why you picked this cache size? Or did it just seem like a reasonable default / size?

Reply:
There was no particular reason for choosing `1000`. You can adjust it if you want.